src/main/kotlin/no/nav/familie/ef/søknad/mapper/kontrakt/BarnMapper.kt

```diff
@@ -35,6 +42,26 @@ object BarnMapper : MapperMedVedlegg<List<Barn>, List<Kontraktbarn>>(BarnaDine)
         }
     }
 
+    fun mapTilDto(barn: List<Kontraktbarn>): List<Barn> {
+        return barn.map {
+            Barn(
+                alder = null,
+                ident = TekstFelt(FødselsnummerTekst.hentTekst(), it.fødselsnummer?.verdi?.verdi ?: ""),
+                fødselsdato = it.fødselTermindato.tilDatoFelt(),
+                harSammeAdresse = BooleanFelt(it.harSkalHaSammeAdresse.label, it.harSkalHaSammeAdresse.verdi),
+                ikkeRegistrertPåSøkersAdresseBeskrivelse = it.ikkeRegistrertPåSøkersAdresseBeskrivelse.tilTekstFelt(),
+                id = null,
```

Comment:
Denne må settes enten her eller i frontend for å "mappe" barnepass til et konkret barn.

Reply:
Ga den en random uuid her

Reply:
Har testet i preprod, og det ser tilstrekkelig ut med en random uuid :)